ui-components/src/helpers/pdfs.ts

```diff
@@ -0,0 +1,14 @@
+import { ListingEvent, ListingEventType } from "@bloom-housing/backend-core/types"
+
+export const cloudinaryPdfFromId = (publicId: string) => {
+  const cloudName = process.env.cloudinaryCloudName || process.env.CLOUDINARY_CLOUD_NAME
+  return `https://res.cloudinary.com/${cloudName}/image/upload/${publicId}.pdf`
+}
```

Comment:
Can you update this to take cloudName as an argument too? I don't think UI-Components should be looking for environment variables.